packages/blitz-rpc/package.json

```diff
@@ -35,18 +35,22 @@
     "supports-color": "8.1.1"
   },
   "peerDependencies": {
+    "@tanstack/query-core": "4.24.4",
+    "@tanstack/react-query": "4.24.4",
```

Comment:
do we have to move this to peerDeps? it's a breaking change

Reply:
No I missed to revert this. `@tanstack/query-core` should stay in peerDeps but `@tanstack/react-query` should go back to a regular dependency